src/main/java/core/basesyntax/strategy/ReturnHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class ReturnHandler implements OperationHandler {
+    private static final String RETURN = "r";
```

Comment:
The constant `RETURN` is not necessary here as the `ReturnHandler` class should inherently deal with 'return' operations. You can remove this constant to simplify the code.